shared/components/theme/more-button.jsx

```diff
@@ -69,7 +73,7 @@ var ThemeMoreButton = React.createClass( {
 						if ( option.url ) {
 							return (
 								<a className="theme__more-button-menu-item popover__menu-item"
-									onMouseOver={ event => {
+									onMouseOver={ function( event ) {
```

Comment:
Isn't this the same as with `=>`? We're still creating a new function on each render, the only difference is in the lack of context binding. Should the mouse-over handler be made a component method?